Makefile.devel

```diff
@@ -15,6 +15,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+PYTHON_VERSION_DIR = $(shell python -c "from __future__ import print_function; \
```

Comment:
Should be able to use the `util/chplenv/chpl_python_version.py` script here too. Makefile.devel is only ever used by the top level makefile which sets CHPL_MAKE_HOME